packages/teams-js/src/public/chat.ts

```diff
@@ -67,20 +74,26 @@ export namespace chat {
    * @beta
    */
   export function openChat(openChatRequest: OpenSingleChatRequest): Promise<void> {
+    const apiVersionTag = getApiVersionTag(chatTelemetryVersionNumber, ApiName.Chat_OpenChat);
+    return openChatHelper(apiVersionTag, openChatRequest);
+  }
+
+  function openChatHelper(apiVersionTag: string, openChatRequest: OpenSingleChatRequest): Promise<void> {
```

Comment:
[AE-MS](https://github.com/AE-MS) commented [on Nov 28, 2023](https://github.com/OfficeDev/microsoft-teams-library-js/pull/2108/files#issuecomment-1831036722)

Do we care to use telemetry to determine how often people call openGroupChat but only pass a single "chatter"?Because as it is now, when someone calls openGroupChat with a single chatter, it will only get recorded as a call to openChat which won't necessarily help us make decisions about how frequently people are calling openGroupChat

Reply:
@AE-MS I think this is the same edge case as we have for openLink and we try to be as accurate as possible. So I create a helper function and move logics to that function to make sure even if `openChat` is called from `openGroupChat`, we could still know that it is from `openGroupChat`.